src/ResourceManagement/EventHub/Microsoft.Azure.Management.EventHub/project.json

```diff
@@ -0,0 +1,57 @@
+{
+  "version": "1.0.1-preview",
+  "description": "Provides developers with libraries to create and manage Namespaces and manage Authorization Rules. Note: This client library is for EventHub under Azure Resource Manager.",
+  "authors": [ "Microsoft" ],
+
+  "packOptions": {
+    "summary": "Provides developers with libraries to create and manage Namespaces ans manage Authorization Rules at both the resource levels. Note: This client library is for EventHub under Azure Resource Manager.",
+    "iconUrl": "http://go.microsoft.com/fwlink/?LinkID=288890",
+    "tags": [ "Microsoft Azure EventHub Management", "EventHub", "EventHub management", "REST HTTP client", "windowsazureofficial" ],
+    "projectUrl": "https://github.com/Azure/azure-sdk-for-net",
+    "licenseUrl": "https://raw.githubusercontent.com/Microsoft/dotnet/master/LICENSE",
+    "requireLicenseAcceptance": true
+  },
+
+  "buildOptions": {
+    "delaySign": true,
+    "publicSign": false,
+    "keyFile": "../../../../tools/MSSharedLibKey.snk"
+  },
+
+  "dependencies": {
+  },
+
+  "frameworks": {
+    "net45": {
+      "dependencies": {
+        "Microsoft.Rest.ClientRuntime.Azure": "[3.1.0,4.0.0)"
```

Comment:
I just picked a service and used their solution. Is there any service I can use as a reference?